.vib/redmine/cypress/cypress/fixtures/users.json

```diff
@@ -0,0 +1,9 @@
+{
+  "newUser": {
+    "userName": "JaneJessica",
+    "password": "someComplicatedPass12345!",
+    "firstName": "Jane Doe Jess",
+    "lastName": "Doe",
+    "email": "thisisanemail"
```

Comment:
Is this a valid email?

Reply:
Made the change requested in a separate comment.